README.md

```diff
@@ -2,3 +2,10 @@ totem-application
 =================
 
 Application that allows to create totems to be included in other pages.
```

Comment:
`totem application` -> `application to creates totem`.  OK.  I know you didn't modify this part but you may explain what's a Totem (or give a screenshot? or both?).  That would be wonderful :-)